### PR TITLE
Added ability to query 24hr ticker price change statistics for all sy…

### PR DESCRIPTION
**Pull request by nomailforme**
Added ability to query 24hr ticker price change statistics for all symbols and to query latest price for single symbol

Signed-off-by: Sai Chaitanya Chitneedi

### CODE REVIEW

File: src/main/java/com/binance/api/client/BinanceApiAsyncRestClient.java [hunk 1]
```diff
@@ -108,13 +108,28 @@
...
    */
   void getAllPrices(BinanceApiCallback<List<TickerPrice>> callback);
+  
+  /**
+   * Get latest price for <code>symbol</code> (asynchronous).
+   * 
+   * @param symbol ticker symbol (e.g. ETHBTC)
+   * @param callback the callback that handles the response
+   */
+   void getSymbolPrice(String symbol , BinanceApiCallback<TickerPrice> callback);
```
Comment (joaopsilva) [inline src/main/java/com/binance/api/client/BinanceApiAsyncRestClient.java]: Can this be called `getPrice`?

File: src/main/java/com/binance/api/client/impl/BinanceApiAsyncRestClientImpl.java [hunk 2]
```diff
@@ -89,11 +89,21 @@ public void getCandlestickBars(String symbol, CandlestickInterval interval, Bina
   public void get24HrPriceStatistics(String symbol, BinanceApiCallback<TickerStatistics> callback) {
     binanceApiService.get24HrPriceStatistics(symbol).enqueue(new BinanceApiCallbackAdapter<>(callback));
   }
+  
+  @Override
+  public void getAll24HrPriceStatistics(BinanceApiCallback<List<TickerStatistics>> callback) {
+	  binanceApiService.getAll24HrPriceStatistics().enqueue(new BinanceApiCallbackAdapter<>(callback));
```
Comment (joaopsilva) [inline src/main/java/com/binance/api/client/impl/BinanceApiAsyncRestClientImpl.java]: Please use 2 whitespaces for indentation.

File: src/main/java/com/binance/api/client/impl/BinanceApiService.java [hunk 3]
```diff
@@ -61,9 +61,15 @@
...
   Call<TickerStatistics> get24HrPriceStatistics(@Query("symbol") String symbol);
+  
+  @GET("/api/v1/ticker/24hr")
+  Call<List<TickerStatistics>> getAll24HrPriceStatistics();
 
   @GET("/api/v1/ticker/allPrices")
   Call<List<TickerPrice>> getLatestPrices();
+  
+  @GET("/api/v3/ticker/price")
+  Call<TickerPrice> getSymbolPrice(@Query("symbol") String symbol);
```
Comment (joaopsilva) [inline src/main/java/com/binance/api/client/impl/BinanceApiService.java]: For consistency, can you please call this `getLatestPrice`?

File: src/main/java/com/binance/api/client/BinanceApiAsyncRestClient.java (hunk 1 above)
Comment (nomailforme) [reply src/main/java/com/binance/api/client/BinanceApiAsyncRestClient.java]: Done

File: src/main/java/com/binance/api/client/impl/BinanceApiAsyncRestClientImpl.java (hunk 2 above)
Comment (nomailforme) [reply src/main/java/com/binance/api/client/impl/BinanceApiAsyncRestClientImpl.java]: Done, But I would have much preferred tab for indentation. Its more standard In my opinion :)

File: src/main/java/com/binance/api/client/impl/BinanceApiService.java (hunk 3 above)
Comment (nomailforme) [reply src/main/java/com/binance/api/client/impl/BinanceApiService.java]: done

**Comment by nomailforme** [comment]
Made changes. Feel free to merge :)

**Comment by joaopsilva** [comment]
Thank you @nomailforme .

**Pull request merged**